monitoring/ceph-mixin/dashboards_out/ceph-cluster-advanced.json

```diff
@@ -0,0 +1,3792 @@
+               }
+            ]
+         },
+         "gridPos": {
+            "h": 6,
+            "w": 24,
+            "x": 0,
+            "y": 46
+         },
+         "id": 38,
+         "links": [ ],
+         "options": {
+            "footer": {
+               "countRows": false,
+               "enablePagination": false,
+               "fields": "",
+               "reducer": [
+                  "sum"
+               ],
+               "show": false
+            },
+            "frameIndex": 1,
+            "showHeader": true
+         },
+         "pluginVersion": "9.4.7",
+         "styles": "",
+         "targets": [
+            {
+               "datasource": "${DS_PROMETHEUS}",
+               "exemplar": false,
+               "expr": "count by (ceph_version)(ceph_osd_metadata{})",
+               "format": "table",
+               "hide": false,
+               "instant": true,
+               "interval": "",
+               "intervalFactor": 1,
+               "legendFormat": "",
+               "range": false,
+               "refId": "A"
+            },
+            {
+               "datasource": "${DS_PROMETHEUS}",
+               "exemplar": false,
+               "expr": "count by (ceph_version)(ceph_mon_metadata{})",
+               "format": "table",
+               "hide": false,
+               "instant": true,
+               "interval": "",
+               "intervalFactor": 1,
+               "legendFormat": "",
+               "range": false,
+               "refId": "B"
+            },
+            {
+               "datasource": "${DS_PROMETHEUS}",
+               "exemplar": false,
+               "expr": "count by (ceph_version)(ceph_mds_metadata{})",
+               "format": "table",
+               "hide": false,
+               "instant": true,
+               "intervalFactor": 1,
+               "legendFormat": "",
+               "range": false,
+               "refId": "C"
+            },
+            {
+               "datasource": "${DS_PROMETHEUS}",
+               "exemplar": false,
+               "expr": "count by (ceph_version)(ceph_rgw_metadata{})",
+               "format": "table",
+               "hide": false,
+               "instant": true,
+               "interval": "",
+               "intervalFactor": 1,
+               "legendFormat": "",
+               "range": false,
+               "refId": "D"
+            },
+            {
+               "datasource": "${DS_PROMETHEUS}",
+               "exemplar": false,
+               "expr": "count by (ceph_version)(ceph_mgr_metadata{})\n",
+               "format": "table",
+               "hide": false,
+               "instant": true,
+               "interval": "",
+               "intervalFactor": 1,
+               "legendFormat": "__auto",
+               "range": false,
+               "refId": "E"
+            }
+         ],
+         "timeFrom": null,
+         "timeShift": null,
+         "title": "Ceph Versions",
```

Comment:
Also, you need to remove this title as this causes an unnecessary layer in the panel and causes the tests to fail because of the duplicate panel name in one single file.